portfolio/src/main/java/com/google/sps/servlets/CommentsHandler.java

```diff
@@ -0,0 +1,70 @@
+package com.google.sps.servlets;
+
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+import com.google.gson.Gson;
+import java.lang.reflect.Field;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Map.Entry;
+
+public class CommentsHandler {
+    private DatastoreService datastore;
+
+    public CommentsHandler() {
+        datastore = DatastoreServiceFactory.getDatastoreService();
+    };
+
+    public String saveComment(Comment comment) {
+        Entity commentEntity = new Entity("Comment");
+        for (Field field: Comment.class.getDeclaredFields()) {
+            try {
+                if (field.get(comment) != null) {
+                    commentEntity.setProperty(field.getName(), field.get(comment));
+                }
+            } catch (IllegalAccessException ex) {
+                System.out.println(ex.getMessage());
```

Comment:
Do we wanna put comment to datastore if we have an exception raised for one of the fields?